products/identity/internal-team/system-recovery-plan/incident-declaration.md

```diff
@@ -0,0 +1,54 @@
+
+## **How to Use This Document**
+
+This document provides guidance on identifying and managing incidents related to key authentication services on VA.gov. Here's how to use it:
+
+1. **Definition and Declaration of an Incident**: Begin by reviewing what constitutes an incident and the process for declaring one. This section explains the criteria for classifying an event as an incident based on user impact.  
+2. **Severity/Priority Levels and Response Times**: Next, determine the severity of the incident. This section helps you prioritize and respond based on the level of impact and the number of affected users.  
+3. **Service Degradation Definitions and User Impact**: Finally, consult this section to understand how to assess the impact using specific monitors and thresholds. This ensures you follow the correct steps to handle the incident effectively.
+
+## **Definition and Declaration of an Incident**
+
+An incident within the scope of the OCTO Identity team is an event that compromises the security, performance, or availability of key authentication services on VA.gov. This includes outages, bugs, or performance issues that significantly impact users. 
+
+**An event should be classified as an incident if the number of users affected on VA.gov has surpassed the limits defined in the User Impact table below.** An incident must be officially declared when its severity reaches the "sev-1" or "P1" level, which indicates a critical impact on key services. 
+
+## **Severity/Priority Levels and Response Times**
+
+All production services for the VA.gov Identity team are considered Tier 1 services. Staging is considered Tier 2\. Everything not production or staging is considered Tier 3\. Outages are prioritized based on their severity which is determined by the number of users affected per hour:
+
+| Service Type | \> 1000 users affected/hour | \> 100 users affected/hour | \> 0 users affected/hour |
+| :---: | :---: | :---: | :---: |
+| **Tier 1**  | **P1** | **P2** | **P3** |
+| **Tier 2**  | **P2** | **P3** | **P4** |
+| **Tier 3** | **P3** | **P4** | **P4** |
+
+## **User Impact**
+
+**User Impact** is defined by the inability of users to complete tasks within a reasonable timeframe. 
+
+**As a rule of thumb, if an outage lasts 15 minutes or more, it likely has affected 1,000 or more users, warranting the declaration of a P1 incident. Outages shorter than 15 minutes typically do not have sufficient impact to justify an incident declaration.**
+
+To understand the impact and severity of an outage, use the following monitors to check how many users are affected:
+
+| Service | Number of users impacted | Monitor showing user impact | Severity Level |
+| :---: | :---: | :---: | :---: |
+| **SSOe** |  If an outage lasts 15 minutes or more, it likely has affected 1,000 or more users, warranting the declaration of a P1 incident. | [SSOe Success Rate Monitor per CSP](https://vagov.ddog-gov.com/monitors/89113?view=spans) | **P1** |
```

Comment:
Since you state the rule of thumb above, I would suggest simplifying this table:
`Number of users impacted` -> `Outage  greater than 15 minutes` 

And just copy that line for `SSOe`, `SiS`, and for `SSOe`, `SiS`, and `STS` in the table below